### PR TITLE
Add examples for outreach

### PR DESCRIPTION
**Pull request by gonuke**
This adds the following info to support the outreach 
- outreach data
- a standard layout
- a markdown page to use the two of them

### CODE REVIEW

File: _data/outreach.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+- date: "2021-12-15"
```
Comment (ngranda) [inline _data/outreach.yml:1]:
```suggestion
- date: "2020-12-15"
```

**Review by bam241 — APPROVED** [review]
LGTM!
but maybe add documentation to explain the workflow to update the `outreach.yml`

**Comment by gonuke** [comment]
@bam241 - I added a document to describe the data files

File: _data/outreach.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  person: Paul Wilson
+  person_dir: pphw
+- date: "2020-12-15"
+  url: https://www.wisbusiness.com/2020/nuclear-energy-could-be-path-to-wisconsins-zero-carbon-future/
+  title: Nuclear energy could be path to Wisconsin’s zero-carbon future
+  venue: WisBusiness
+  venue_url: https://www.wisbusiness.com
+  person: Paul Wilson
+  person_dir: pphw
+- date: 2019-10
```
Comment (bam241) [inline _data/outreach.yml:15]: format is not consistant with the other dates.

Does it matter ?

**Review by bam241 — APPROVED** [review]
just that one comment, otherwise LGTM!

File: _data/data.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Examples include the [Community Events](community/events.md), and many of the
+lists of resources in the manual.
+
+### Presentation data
+
+These data files contain information that can be used to provide a list of
+presentations, appearances, or outreach publications, including hyperlinks.
+
+The data is a list of dictionaries, each consisting of:
```
Comment (nuclearkatie) [inline _data/data.md:61]: Do we want to include an optional image? I recently added a [similar section](https://nuclearkatie.github.io/research-stuff/presentation) to my website, and I think having an image (if available) looks quite nice

File: _data/outreach.yml (hunk 2 above)
Comment (gonuke) [reply _data/outreach.yml:15]: No. There were actually a couple of dates in that October using the same resource

**Comment by bam241** [comment]
LGTM!
merging!

File: _data/data.md (hunk 3 above)
Comment (bam241) [reply _data/data.md:61]: this would be addressed in an other PR

**Pull request merged**